seccomp/mount.go

```diff
@@ -528,6 +620,13 @@ func (m *mountSyscallInfo) createRemountPayload(
 func (m *mountSyscallInfo) processBindMount(
 	mip *mountInfoParser) (*sysResponse, error) {
 
+	// Notice that we are not adjusting mount attributes to deal with chroot'ed
```

Comment:
I don't fully understand this. Let's talk.

Reply:
Bind-mounts are (apparently) not properly managed by kernel in chroot'ed contexts. See below that bind-mounts done within the chroot jail are only displayed outside the chroot ...

My point here is that we should keep consistency with what the kernel is currently doing, so that's why i'm not calling rootAdjust() method.

```
rmolina@dev-vm1:~$ docker run --privileged -it --rm --name ubuntu ubuntu
root@afa472988d2d:/#

root@afa472988d2d:/# pwd
/
root@afa472988d2d:/# mkdir rootfs
root@afa472988d2d:/# cp -a /usr rootfs/
root@afa472988d2d:/# cp -a /lib rootfs/
root@afa472988d2d:/# cp -a /lib64 rootfs/
root@afa472988d2d:/# cp -a /bin rootfs/

root@afa472988d2d:/# chroot /rootfs
#
# mount
mount: /proc: mount point does not exist.

# mkdir /proc
# mount -t proc proc /proc

# findmnt
TARGET SOURCE FSTYPE OPTIONS
/proc  proc   proc   rw,relatime
#

# mkdir -p /a /a/b /x /x/y
# mount -o bind /a /x
#
# findmnt
TARGET SOURCE FSTYPE OPTIONS
/proc  proc   proc   rw,relatime
#
```

<-- Outside chroot jail (back to container):

```
root@afa472988d2d:/# findmnt
TARGET                                SOURCE                                                                           FSTYPE  OPTIONS
/                                     overlay                                                                          overlay rw,relatime,lowerdir=/var/lib/docker/overlay2/l/6ZJX5TKOFYRQRL2O3E7R2I65S
|-/rootfs/proc                        proc                                                                             proc    rw,relatime
|-/rootfs/x                           overlay[/rootfs/a]                                                               overlay rw,relatime,lowerdir=/var/lib/docker/overlay2/l/6ZJX5TKOFYRQRL2O3E7R2I65S
```

Reply:
I ran this experiment again. Kernel is indeed making the bind-mount in the expected jailed context, but for some reason 'findmnt' is not displaying them. However, /proc/pid/mountinfo has the right information, so looks like a 'findmnt' bug.

My point: we need to handle bind-mounts within chroot jails.